ktor-http/ktor-http-cio/jvm/src/io/ktor/server/cio/backend/ServerPipeline.kt

```diff
@@ -87,7 +87,7 @@ public fun CoroutineScope.startServerConnectionPipeline(
 
             val response = ByteChannel()
 
-            val transferEncoding = request.headers["Transfer-Encoding"]
+            val transferEncoding = request.headers["Transfer-Encoding"]?.trim()
```

Comment:
Is this covered by `testChunkedWithContentLength`?

Reply:
It should be removed, accidentally cherry picked. Thanks for the notice